### DISCUSSION TITLE
nuclei reported an error when scanning

### DISCUSSION THREAD

**Discussion by 867810770** [projectdiscovery/nuclei-templates#4904] [Q&A]
When I used nuclei to scan 100 url addresses, I encountered an error message. The detailed error message is shown in the figure.
![微信图片_20220724231041](https://user-images.githubusercontent.com/42743655/180653704-6f42f75c-4dfc-46c3-bb1b-bf127877a11e.png)

**Comment by daffainfo** [comment]
@867810770  Can you share your command?
